practice_2/StarWarsApp/cells/HeroCell.swift

```diff
@@ -0,0 +1,18 @@
+import UIKit
+
+class HeroCell: UITableViewCell {
+    static let identifier = "heroCell"
+    @IBOutlet weak var heroNameLabel: UILabel!
```

Comment:
IBOutlets should be private to avoid leaking UIKit to higher layers

Reply:
Done. Thanks.